<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsOWSSourceSelectBase</class>
 <widget class="QDialog" name="QgsOWSSourceSelectBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>743</width>
    <height>689</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Layer(s) from a Server</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>.</normaloff>.</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Help</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="mStatusLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Ready</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="mTabWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="mLayersTab">
      <attribute name="title">
       <string>Layers</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QComboBox" name="mConnectionsComboBox"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="mConnectButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Connect to selected service</string>
           </property>
           <property name="text">
            <string>C&amp;onnect</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="mNewButton">
           <property name="toolTip">
            <string>Create a new service connection</string>
           </property>
           <property name="text">
            <string>&amp;New</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="mEditButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Edit selected service connection</string>
           </property>
           <property name="text">
            <string>Edit</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="mDeleteButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Remove connection to selected service</string>
           </property>
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_1">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>82</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="mLoadButton">
           <property name="toolTip">
            <string>Load connections from file</string>
           </property>
           <property name="text">
            <string>Load</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="mSaveButton">
           <property name="toolTip">
            <string>Save connections to file</string>
           </property>
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="mAddDefaultButton">
           <property name="statusTip">
            <string>Adds a few example WMS servers</string>
           </property>
           <property name="toolTip">
            <string comment="Adds several example WMS servers to the list"/>
           </property>
           <property name="text">
            <string>Add Default Servers</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTreeWidget" name="mLayersTreeWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::MultiSelection</enum>
         </property>
         <property name="allColumnsShowFocus">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string>ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Title</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Abstract</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="mTimeWidget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="mTimeLabel">
            <property name="text">
             <string>Time</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="mTimeComboBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="mCRSWidget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="mCRSLabel">
            <property name="text">
             <string>Coordinate Reference System</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="mSelectedCRSLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Selected Coordinate Reference System</string>
            </property>
            <property name="buddy">
             <cstring>mChangeCRSButton</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="mChangeCRSButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Changeâ€¦</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="mFormatWidget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="mFormatLabel">
            <property name="text">
             <string>Format</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="mFormatComboBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="mCacheWidget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="mCacheLabel">
            <property name="text">
             <string>Cache</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="mCacheComboBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Cache preference

Always cache: load from cache, even if it expired

Prefer cache: load from cache if available, otherwise load from network. Note that this can return possibly stale (but not expired) items from cache

Prefer network: default value; load from the network if the cached entry is older than the network entry

Always network: always load from network and do not check if the cache has a valid entry (similar to the &quot;Reload&quot; feature in browsers)
</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="mSpatialExtentBoxWidget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QgsExtentGroupBox" name="mSpatialExtentBox">
            <property name="title">
             <string>Extent (current: none)</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <property name="collapsed" stdset="0">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="mLayerOrderTab">
      <attribute name="title">
       <string>Layer Order</string>
      </attribute>
      <layout class="QGridLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="mLayerUpButton">
         <property name="toolTip">
          <string>Move selected layer UP</string>
         </property>
         <property name="text">
          <string>Up</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="mLayerDownButton">
         <property name="toolTip">
          <string>Move selected layer DOWN</string>
         </property>
         <property name="text">
          <string>Down</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>391</width>
           <height>30</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QTreeWidget" name="mLayerOrderTreeWidget">
         <property name="columnCount">
          <number>2</number>
         </property>
         <column>
          <property name="text">
           <string>Layer</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Style</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="mTilesetsTab">
      <attribute name="title">
       <string>Tilesets</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QTableWidget" name="mTilesetsTableWidget">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="cornerButtonEnabled">
          <bool>false</bool>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Layers</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Styles</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Size</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Format</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>CRS</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QgsCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>qgscollapsiblegroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsExtentGroupBox</class>
   <extends>QgsCollapsibleGroupBox</extends>
   <header>qgsextentgroupbox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mTabWidget</tabstop>
  <tabstop>mConnectionsComboBox</tabstop>
  <tabstop>mConnectButton</tabstop>
  <tabstop>mNewButton</tabstop>
  <tabstop>mEditButton</tabstop>
  <tabstop>mDeleteButton</tabstop>
  <tabstop>mLoadButton</tabstop>
  <tabstop>mSaveButton</tabstop>
  <tabstop>mAddDefaultButton</tabstop>
  <tabstop>mLayersTreeWidget</tabstop>
  <tabstop>mTimeComboBox</tabstop>
  <tabstop>mChangeCRSButton</tabstop>
  <tabstop>mFormatComboBox</tabstop>
  <tabstop>mCacheComboBox</tabstop>
  <tabstop>mLayerUpButton</tabstop>
  <tabstop>mLayerDownButton</tabstop>
  <tabstop>mLayerOrderTreeWidget</tabstop>
  <tabstop>mTilesetsTableWidget</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QgsOWSSourceSelectBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>518</x>
     <y>510</y>
    </hint>
    <hint type="destinationlabel">
     <x>551</x>
     <y>370</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
